### DISCUSSION TITLE
Workspace vs single package

### DISCUSSION THREAD

**Discussion by Matthew-Skeer** [leptos-rs/leptos#2264] [Q&A] [answered]
Good Afternoon, 

I understand that cargo-leptos supports both single-package and workspace setup of projects, but what are the pro/cons of workspace vs single package. Is there any particular use case. or is it just for code seperation.

**Comment by gbj** [comment, marked as answer]
No real differences beyond personal preference! Some people prefer the workspace setup if they have server-only code and they don't want to `#[cfg]` gate things, or if they're integrating with an existing project, or want to reuse pieces of their logic. The single-crate setup works fine for most use cases.

**Reply by Matthew-Skeer** [reply]
cheers
